lib/mail_room/delivery/postback.rb

```diff
@@ -101,6 +101,17 @@ def config_request_jwt_auth(request)
         request.headers[@delivery_options.jwt.header] = @delivery_options.jwt.token
       end
 
+      def config_token_auth(connection)
+        if defined?(connection.token_auth)
+          connection.token_auth @delivery_options.token
```

Comment:
Would you be able to comment here on why this condition exists (the faraday version differences)?

Reply:
Done